### ISSUE TITLE
Still having errors when making submissions public by organizers

### ISSUE THREAD

**Issue by vponcelo** [codalab/codalab-competitions#1585]
Hi,

These are the previous issues from **the administration panel** related to the errors when making submissions public:

https://github.com/codalab/codalab-competitions/issues/1569
https://github.com/codalab/codalab-competitions/issues/1570

**We realized it is working on the test server but not in the general server.**
We tried it again in several browsers: Google Chrome, Firefox and Safari:

<img width="456" alt="error" src="https://cloud.githubusercontent.com/assets/3542257/17559397/624105e2-5f1e-11e6-8f57-8937d75cdf89.png">

<img width="428" alt="error2" src="https://cloud.githubusercontent.com/assets/3542257/17559432/85ff874c-5f1e-11e6-9ec9-9d514225d879.png">

<img width="447" alt="errorsafari" src="https://cloud.githubusercontent.com/assets/3542257/17559490/b3cf167e-5f1e-11e6-89b3-f6d23a85bd81.png">

**Comment by FlavioAlexander** [comment]
@vponcelo Fix is already in place, i haven't pushed the changes to production. Will take care of that in the next few hours. I will let you know once this is done

**Comment by FlavioAlexander** [comment]
@vponcelo Changes are in the production environment now

**Comment by vponcelo** [comment]
@FlavioAlexander Thank you again. 
It seems the system allows now to make participant's submissions public without raising any error. However, they do not appear in the public submission leaderboard:
https://competitions.codalab.org/competitions/public_submissions/10751
(even for the learning phase that is set to "default" and not to "hidden")
I know you are already working on this, but this is just in case to let you know.

**Comment by FlavioAlexander** [comment]
@vponcelo You need to publish your submission in order to make it public. Submission are not forced to be public, you can do this by clicking `Make your submission public`. Your `learning` phase should have some public submissions. It seems like you haven't publish any submission the `Final Evaluation` phase

**Comment by vponcelo** [comment]
@FlavioAlexander thanks a lot again. I can see now those submissions I made public in the public submission leaderboard. For the evaluation phase I will make them public tomorrow, following our competition schedule.
Best,

**Comment by FlavioAlexander** [comment]
@vponcelo I have created #1589 to follow up with this discussion in the future. Add any comment/recommendation you consider important in the new issue
